src/oop/programs_with_multiple_classes/hard/q2/TestFort.java

```diff
@@ -0,0 +1,13 @@
+package oop.programs_with_multiple_classes.hard.q2;
+
+public class TestFort {
+    public static void main(String[] args) {
```

Comment:
Can we make this a method instead of the main method? Like, have three methods which each takes the values to pass to the constructor and then return the constructed object?